### DISCUSSION TITLE
[Feature Request] some Important function

### DISCUSSION THREAD

**Discussion by p82183645** [Zrips/CMI#1863] [Feature Requests]
1. add /cmi data , Change item data
Uses: Convenient to make material package item

Example: DIAMOND_SWORD:5
/cmi data 6
change
 DIAMOND_SWORD:6

2. add Maximum enchantment limit
Example: Efficiency III
The server does not appear to exceed the efficiency III

3. armor_stand edit gui , add remove Button ( Sometimes armor can't be removed )

4. /cmi reload , can reload SavedItems.yml 
Manual editing is more convenient
or add /cmi SavedItems reload

5. Kit condition , add play time condition
Example: This player has been playing for 7 days in total to receive this kit.

thank , please add
Especially 1 and 4
These two functions are very important

**Comment by mrfloris** [comment]
Duplicate @ https://github.com/Zrips/CMI/issues/1823

**Comment by p82183645** [comment]
any add?
